docs/src/testing/chains.md

```diff
@@ -1,10 +1,10 @@
 ## Increase the block height 
 
-`add_blocks` can be used to help achieve a desired block height. Useful for when you want to do any testing regarding
+`produce_blocks` can be used to help achieve a desired block height. Useful for when you want to do any testing regarding
 transaction maturity.
```

Comment:
I've applied the suggestion manually since this was outdated, the line breakage wouldn't cause issues in the book since md only breaks paragraphs if you have a double line break, afaics.

Reply:
hehe my macro does the same, and adds John a lot of work when writing docs lol